pkg/genericapiserver/config.go

```diff
@@ -325,15 +341,8 @@ func (c Config) New() (*GenericAPIServer, error) {
 		openAPIDefinitions:     c.OpenAPIDefinitions,
 	}
 
-	if c.RestfulContainer != nil {
-		s.HandlerContainer = c.RestfulContainer
-	} else {
-		s.HandlerContainer = NewHandlerContainer(http.NewServeMux(), c.Serializer)
-	}
-	// Use CurlyRouter to be able to use regular expressions in paths. Regular expressions are required in paths for example for proxy (where the path is proxy/{kind}/{name}/{*})
-	s.HandlerContainer.Router(restful.CurlyRouter{})
-	s.Mux = apiserver.NewPathRecorderMux(s.HandlerContainer.ServeMux)
-	apiserver.InstallServiceErrorHandler(s.Serializer, s.HandlerContainer)
+	s.ProtectedContainer = mux.NewAPIContainer(http.NewServeMux(), c.Serializer)
```

Comment:
> Two of what? origin has two containers.

As I recall, it was set up to be one inside and one outside of authn/authz?  I don't think I care about doing that.  I think I'd rather have one chain and everything goes through authn/authz. 

@liggitt Do you remember why we didn't do that?  If no token is presented, the check should be pretty cheap since we aren't making any external requests.